firmware/arduino/chessboard/chessboard.ino

```diff
@@ -39,7 +26,22 @@ int yMotor[4] = {MOTOR2_STEP_PIN, MOTOR2_DIR, MOTOR2_MS1, MOTOR2_MS2};
 // Distance Definitions
 #define MILLIMETERS_PER_SQUARE 63
 #define STEPS_PER_MILLIMETER 5
+#define HOME_OFFSET 100
```

Comment:
Sorry for the noob question, but what does this variable represent? Offset from bottom left corner of board?